src/mono/netcore/sample/wasm/browser/Makefile

```diff
@@ -0,0 +1,19 @@
+TOP=../../../../../..
+
+DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
+
+CONFIG?=Release
+
+all: build
+
+build:
+	$(TOP)/.dotnet/dotnet build $(DOTNET_Q_ARGS) /p:TargetArchitecture=wasm /p:TargetOS=Browser /p:Configuration=$(CONFIG) WasmSample.csproj
+
+clean:
+	rm -rf bin
+
+run:
+	cd bin/$(CONFIG)/publish && python server.py
+
+runtimepack:
+	EMSDK_PATH=$(abspath $(TOP)/src/mono/wasm/emsdk) $(TOP)/build.sh -c $(CONFIG) -os Browser -arch wasm -subset Mono+Libs
```

Comment:
I don't think we should keep this, it doesn't respect the EMSDK_PATH set by an env var (I know you copied it from the console sample).